tests/ballerina-unit-test/src/test/resources/test-src/workers/invalid-async-send-in-fork.bal

```diff
@@ -0,0 +1,23 @@
+public function main() {
+    fork {
+        worker w1{
+            int i = 20;
+        }
+
+        worker w2 {
+            int j = 25;
+            int sum = 0;
+            fork {
+                worker w3 {
+                    foreach var i in 1... 10 {
+                        sum = sum + i;
+                        i -> w4;
+                    }
+                }
+                worker w4 {
+                    int k = <- w3;
+                }
+            }
+        }
+    }
+}
```

Comment:
Hey, Shall we keep a new line at the end of the file as a standard? Because this is being pointed on linting rules in GitHub.